testsuite/shared/enable-elytron.cli

```diff
@@ -0,0 +1,40 @@
+embed-server --server-config=standalone.xml
```

Comment:
is there any overlap between this .cli file and the modifly-elytron-config.cli in elytron test module?